docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[log-correlation]]
+== Log Correlation
+
+The Go agent provides integrations for popular logging frameworks, which take care of
+injecting trace ID fields into your application's log records. You can find a list of
+the supported logging frameworks under <<supported-tech-logging, supported technologies>>.
+
+If your favorite logging framework is not already supported, there are two other options:
+
+* Open a feature request, or contribute code, for additional support, as described in <<contributing>>.
+* Manually inject trace IDs into log records, as described below in <<log-correlation-manual>>.
+
+Regardless of how you integrate APM with logging, you can use {filebeat-ref}[Filebeat] to
+send your logs to Elasticsearch, in order to correlate your traces and logs and link from
+APM to the {kibana-ref}/xpack-logs.html[Logs app].
+
+[float]
+[[log-correlation-manual]]
+=== Manual log correlation
+
+If the agent-provided logging integrations are not suitable or not available for your
+application, then you can use the agent's <<api, API>> to inject trace IDs manually.
+There are two main approaches you can take, depending on whether you are using structured
+or unstructured logging.
+
+[float]
+[[log-correlation-manual-structured]]
+==== Manual log correlation (structured)
+
+For correlating structured logs with traces, the following fields should be added to your logs:
+
+ - trace.id
```

Comment:
Any reason why these aren't formatted as code? For example, `trace.id` like you do [here](https://github.com/elastic/apm-agent-go/pull/635/files#diff-9b5a72acb7c2dd95c30074560cccc3beR199).

Reply:
None at all, will fix.